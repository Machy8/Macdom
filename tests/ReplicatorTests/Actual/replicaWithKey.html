@Whisky Whisky number |
@Beer Beer number |

Whisky one <br>
Beer one <br>

div
	Whisky two <br>
	Beer three <br>

/@Whisky
Whisky text
<br>
/@Beer
Beer text
