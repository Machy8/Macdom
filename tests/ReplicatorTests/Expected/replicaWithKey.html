Whisky number one 
<br>
Beer number one 
<br>
<div>
	Whisky number two 
	<br>
	Beer number three 
	<br>
</div>
Whisky text
<br>
Beer text
